packages/blockchain-api/src/events/EscrowReceived.ts

```diff
@@ -7,8 +7,8 @@ import { Contracts } from '../utils'
 export class EscrowReceived extends TransactionType {
   matches(transaction: Transaction): boolean {
     return (
-      transaction.transfers.length === 1 &&
-      transaction.transfers.containsTransferFrom(Contracts.Escrow)
+      transaction.transfers.isEscrowReceivedToEOA(Contracts.Escrow) ||
```

Comment:
`isEscrowReceivedToEOA` and `isEscrowReceivedToMTW` are very specific to the `EscrowReceived` class, so it doesn't make sense to move them to `TransfersNavigator` as no other code would reuse them anyway. Instead we could move them here and get rid of the parameter as both methods would be able to get the address directly.